src/d3a/models/strategy/pv.py

```diff
@@ -98,8 +98,7 @@ def area_reconfigure_event(self, **kwargs):
             self.panel_count = kwargs['panel_count']
         if key_in_dict_and_not_none(kwargs, 'max_panel_power_W'):
             self.max_panel_power_W = kwargs['max_panel_power_W']
-
-        self.produced_energy_forecast_kWh()
+            self.produced_energy_forecast_kWh()
```

Comment:
Are you certain ? This will have the effect that the produced_energy_forecast will not be updated if e.g. the panel_count parameter is modified, which does not look right to my eyes

Reply:
Ah yes, you are right, i didnt consider the panel count as an energy related parameter. But it certainly is.